frontend_build/src/extract_$trs.js

```diff
@@ -19,7 +43,36 @@ extract$trs.prototype.apply = function(compiler) {
   compiler.plugin('emit', function(compilation, callback) {
     var messageExport = {};
     var nameSpaces = [];
-
+    function registerFoundMessages(messageNameSpace, messages, module) {
+      if (messageNameSpace) {
+        // Warn about duplicate nameSpaces *within* a bundle (no way to warn across).
+        if (nameSpaces.indexOf(messageNameSpace) !== -1) {
+          logging.error('Duplicate namespace ' + messageNameSpace + ' found in ' + module.resource);
```

Comment:
It's still a namespace, it's just that we use the Vue `name` property for the namespace in .vue files.